racket/src/zuo/zuo-doc/zuo-build.scrbl

```diff
@@ -494,18 +494,21 @@ such a target to one where @racket[args] are the argument when the
 target is built.}
 
 @deftogether[(
-@defproc[(file-sha1 [file path-string?] [token (or/c token? #f)]) sha1?]
-@defproc[(sha1? [v any/c]) booelan?]
+@defproc[(file-sha256 [file path-string?] [token (or/c token? #f)]) sha256?]
+@defproc[(sha256? [v any/c]) booelan?]
+@defthing[sha256-length integer? #:value 64]
 )]{
 
-The @racket[file-sha1] procedure returns the SHA-1 hash of the content
-of @racket[file], or it returns @racket[no-sha1] if @racket[file] does
+The @racket[file-sha256] procedure returns the SHA-256 hash of the content
+of @racket[file], or it returns @racket[no-sha256] if @racket[file] does
 not exist.
 
-The @racket[sha1?] predicate recognizes values that are either a
-40-character string or @racket[no-sha1].}
+The @racket[sha256?] predicate recognizes values that are either a
+64-character string (thus, @racket[sha256-length]) or @racket[no-sha256].
```

Comment:
This doesn't cover the concatenation case:
```racket
(define (sha256? s)
  (or (and (string? s)
           (or (= (string-length s) sha256-length)
               (string=? s no-sha256)
               ;; also allow a concatenation of SHA-256s for multi-file targets
               (and (>= (string-length s) sha256-length)
                    (= 0 (modulo (string-length s) sha256-length)))))
      (eq? s phony-sha256)))
```
Should it, or is that intended to be an implementation detail, like `phony-sha256`?

Reply:
The documentation should be repaired/updated to reflect the implementation.

Reply:
I've pushed an update.